requirements.txt

```diff
@@ -31,6 +31,7 @@ art
 fschat==0.2.36
 gradio==3.50.2
 tensorboard
+mosaicml-streaming
```

Comment:
[`question`](https://conventionalcomments.org/): `mosaicml-streaming` should be an optional dependency. Is this the right way of adding it in this capacity?

Reply:
I'm okay with it being a required dependency. If it causes issues down the line we can make it optional then. Hoping to keep things simpler.

Reply:
Could this version be locked?

Reply:
Addressed by `976bc13`.